### ISSUE TITLE
Obsolete MarkupSafe version causes building error

### ISSUE THREAD

**Issue by robertopreste** [elixir-europe/proTES#52]
**Describe the bug**
The version of MarkupSafe currently listed in the `requirements.txt` file (v.1.0) uses a deprecated module from `setuptools`, and this causes an `ImportError` that breaks the building of proTES.

**To Reproduce**
1. `docker build .` in the root of this repo
2. error issued while installing requirements

**Expected behavior**
The Docker image building should run smoothly.

**Screenshots**
```shell
Collecting MarkupSafe==1.0
  Downloading MarkupSafe-1.0.tar.gz (14 kB)
    ERROR: Command errored out with exit status 1:
     command: /usr/local/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-rvrehuys/MarkupSafe/setup.py'"'"'; __file__='"'"'/tmp/pip-install-rvrehuys/MarkupSafe/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-install-rvrehuys/MarkupSafe/pip-egg-info
         cwd: /tmp/pip-install-rvrehuys/MarkupSafe/
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-rvrehuys/MarkupSafe/setup.py", line 6, in <module>
        from setuptools import setup, Extension, Feature
    ImportError: cannot import name 'Feature'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
The command '/bin/sh -c cd /app   && pip install -r requirements.txt   && cd /' returned a non-zero code: 1
```

**Software**
Add versions of relevant software, e.g.
 - macOS 10.15.4
 - Docker 19.03.8

**Additional context**
The issue can be fixed quite easily, simply replacing `MarkupSafe==1.0` with `MarkupSafe==1.1.1` in the requirements file should do the job.

**Comment by uniqueg** [comment]
Thanks for the detailed bug report @robertopreste, much appreciated. Fix is on the way

**Comment by uniqueg** [comment]
Closed by #53
